symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeDependents.vue

```diff
@@ -0,0 +1,258 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <edit-employee-layout :employee-id="empNumber">
+    <save-dependent
+      v-if="showSaveModal"
+      :http="http"
+      @close="onSaveModalClose"
+    ></save-dependent>
+    <edit-dependent
+      v-if="showEditModal"
+      :http="http"
+      :data="editModalState"
+      @close="onEditModalClose"
+    ></edit-dependent>
+    <div class="orangehrm-horizontal-padding orangehrm-top-padding">
+      <profile-action-header @click="onClickAdd">
+        Assigned Dependents
+      </profile-action-header>
+      <oxd-divider />
+    </div>
+    <div>
+      <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+        <div v-if="checkedItems.length > 0">
+          <oxd-text tag="span">
+            {{ itemsSelectedText }}
+          </oxd-text>
+          <oxd-button
+            label="Delete Selected"
+            iconName="trash-fill"
+            displayType="label-danger"
+            @click="onClickDeleteSelected"
+            class="orangehrm-horizontal-margin"
+          />
+        </div>
+        <oxd-text tag="span" v-else>{{ itemsCountText }}</oxd-text>
+      </div>
+    </div>
+    <div class="orangehrm-container">
+      <oxd-card-table
+        :headers="headers"
+        :items="items?.data"
```

Comment:
is it right ?

Reply:
`items?.data` ? I use the elvis operator in case the paginator cannot resolve the items and fails mid API request. it should prevent errors.

Reply:
ahh I never used `elvis operator` in json data :(

Reply:
It's Es next syntax. (we can use because of babel) very helpful when writing shorthand codes. :smiley: